app/jobs/finalize_import_job.rb

```diff
@@ -0,0 +1,37 @@
+class FinalizeImportJob < ActiveJob::Base
```

Comment:
This should inferit from [`ApplicationJob`](https://github.com/artsy/bearden/blob/master/app/jobs/application_job.rb).